SimPEG/Directives.py

```diff
@@ -144,6 +144,35 @@ def endIter(self):
             if self.debug: print 'BetaSchedule is cooling Beta. Iteration: %d' % self.opt.iter
             self.invProb.beta /= self.coolingFactor
 
+#class BetaSchedule_PGN_CG(InversionDirective):
```

Comment:
can this inherit the BetaSchedule directive? Then it will have the cooling factor and rate by default - it is an extension on the BetaSchedule?